### PR TITLE
Issue/daily menu fetch

### PR DESCRIPTION
**Pull request by Stefan770**

### CODE REVIEW

File: Exebite.GoogleSheetAPI/Common/Constants.cs [hunk 1]
```diff
@@ -20,12 +20,27 @@ public class Constants
...
-        /// Name of the Teglas restaurant
+        /// Name of the Hey Day Restaurant
         /// </summary>
-        public const string TEGLAS_NAME = "Teglas";
+        public const string HEY_DAY_NAME = "Hey Day";
+
+        /// <summary>
+        /// Name of the Parrilla restaurant
+        /// </summary>
+        public const string PARRILLA_NAME = "Parilla";
```
Comment (stanislavgrujic) [inline Exebite.GoogleSheetAPI/Common/Constants.cs:33]: Spelling - is it Parrilla or Parilla?

File: Exebite.GoogleSheetAPI/Common/Constants.cs [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
         /// App name used to identify this app to google servers.
         /// </summary>
         public const string APP_NAME = "Exebite Sheet Overseer";
 
         /// <summary>
-        /// Name of Hedone restaurant
+        /// Name of Topli Obrok restaurant
         /// </summary>
-        public const string HEDONE_NAME = "Hedone";
+        public const string TOPLI_OPBROK_NAME = "Topli Obrok";
```
Comment (stanislavgrujic) [inline Exebite.GoogleSheetAPI/Common/Constants.cs:13]: Spelling OPBROK - OBROK

File: FeatureTestingConsole/App.cs [hunk 3]
```diff
@@ -157,91 +66,37 @@ private void SeedRestaurant()
         {
             _restaurantCommandRepository.Insert(new RestaurantInsertModel()
             {
-                Name = "Lipa restaurant"
+                Name = "Pod Lipom"
```
Comment (stanislavgrujic) [inline FeatureTestingConsole/App.cs:69]: Why not use constants which you have added?

File: Exebite.Common/RestaurantConstants.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+        /// <summary>
+        /// Name of the Mimas restaurant
+        /// </summary>
+        public const string MIMAS_NAME = "Mimas";
+
+        /// <summary>
+        /// Name of the Serpica restaurant
+        /// </summary>
+        public const string SERPICA_NAME = "Serpica";
```
Comment (itesanovic) [inline Exebite.Common/RestaurantConstants.cs:38]: Should we add Serbian letters? So this would be "Šerpica".

File: FeatureTestingConsole/App.cs [hunk 5]
```diff
@@ -52,51 +53,51 @@ private void SeedLocation()
...
-                Address = "Vojvode stepe 50"
+                Address = "Vojvode Stepe 50"
             });
 
             _locationCommandRepo.Insert(new LocationInsertModel()
             {
-                Name = "Execom JD",
-                Address = "Jovana ducica 50"
+                Name = "Execom MM",
+                Address = "Đorđa Rajkovica 2"
```
Comment (itesanovic) [inline FeatureTestingConsole/App.cs:62]: Spelling error with "Rajkovića".

**Pull request merged**